docs: update testium docs with testium-driver-wd 3.x changes (#28)
testiumjs/testiumjs.github.io: api/wd/index.md

});
```

### `browser.getConsoleLogs([logLevel: string]): Promise<Object>`

Returns all log events for `logLevel` `all`(default) `log`,`warn`, `error`, `debug` since the last time this method was called.
**Warning:** Each browser implements this differently against the WebDriver spec.

```js
const errorLogs = browser.getConsoleLogs('error').then(errorLogs => { /* ... */});
```

### `browser.getScreenshot(): Promise<string>`

Returns screenshot as a base64 encoded PNG.

## Navigation

> ### `browser.getHeader(name: string): Promise<any>`

Reviewer: i assume this is really something like `: Promise<string | string[]>` - can we check if this is correct and put the real type?

Returns the value of the response header with the provided name.
Header names should be provided in lowercase.

> ### `browser.getHeaders(): Promise<Object>`

Reviewer: ditto

assert.match(/^https:/, browser.getUrl());
```

### browser.getUrlObject(): Promise<URL>
<span class="new-in">Added in: testium-driver-wd v3.1.0</span>

Returns a [WHATWG URL instance](https://nodejs.org/dist/latest-v13.x/docs/api/url.html#url_the_whatwg_url_api) of the current url.

### browser.getStatusCode(): Promise<number>

Returns the response status code for the current page.

> ### browser.loadPage(url: string[, options?: loadPageOpts]): Promise<void>

Reviewer: no need for the `[...]` - that's what the `?` means

Author: thanks .. I oversaw this one

});
```

> ### `browser.loadPage(url: string[, options: Object]): Promise<void>`

Reviewer: ```suggestion
### `browser.loadPage(url: string, options?: LoadPageOpts): Promise<void>`
```

Reviewer: I'm wondering if we should be more accruate with these types and note that really these return `: Browser & Promise<x>` to show that they're magic chain objects, or if that's needlessly confusing....

  * A RegExp that the status code (as a string) should match
  * A Function which takes the status code and returns `true` or `false` if it is acceptable

> ### `browser.navigateTo(url: string, options: Object): Promise<void>` (deprecated)

Reviewer: ```suggestion
### `browser.navigateTo(url: string, options?: NavigateToOpts): Promise<void>` (deprecated)
```

  .waitForUrl('/products', { start: 30, count: 15 });
```

## Elements

### `browser.clickOn(cssSelector)`

> Calls `clickOn()` on the element found by the given `cssSelector`.

Reviewer: ```suggestion
Calls `click()` on the Element found by the given `cssSelector`.
```

I like the capitalization because it shows that it's operating on an instance of our Element class, but the correct method name for that is `.click()`

```js
browser.clickOn('.button');
```

<span class="new-in">Changed in: testium-driver-wd v3.0.0</span>
The selector passed into `clickOn()` must match **only one unique** element in the page. 
Otherwise `onClick()` will throw. Use `clickOnAll()` for multiple elements.

### `browser.clickOnAll(cssSelector)`
<span class="new-in">Added in: testium-driver-wd v3.0.0</span>

> Calls `clickOnAll()` on all element found by the given `cssSelector`.

Reviewer: ```suggestion
Calls `.click()` on each Element found by the given `cssSelector`.
```

browser.assertElementExists('.user-name');
```

### `browser.assertElementDoesntExist(selector)`

Asserts that the element at `selector` doesn't exist.
Returns the element.

Inverse of `assertElementExists`.

> ### `browser.assertElementsNumber(selector, number)`  **[DEPRECATED]**

Reviewer: I'm not sure it's terribly important that we deprecate this

Asserts that `selector` matches exactly `number` elements.

### `browser.assertElementsNumber(selector, { min: number, max: number, equal: number })`
<span class="new-in">Added in: testium-driver-wd v3.0.0</span>

Asserts that `selector` matches element numbers:

> - `min`:  the least `min` amount of elements

Reviewer: ```suggestion
- `min`:  at least `min` amount of elements
```

});
```

## Cookies

Cookie objects have the following general structure:

```json
{
  "name":  "<name>",
  "value":  "<value>",
  "path": "/",
  "domain": "<currentPageDomain>",
  "expiry": "<endOfCurrentSession>",

>   "secure": false

Reviewer: I think the old syntax was more suggestive of those values being defaults, since that's the JS destructure syntax for defaults

Author: hmm I'm not  sure. I reverted it to before

  .a11yAudit({ ignore: ['color-contrast', 'meta-viewport'] })
  .then(issues => deepEqual([], issues))
```

### `assertAccessibilityScore(minScore: number[, skipAudits: Array<string>])`

> Runs Lighthouse 5.6 `accessibility` audit.

Reviewer: Putting the version number everywhere feels like it's going to tie us to lots of updates every time the version changes.  Is it important to include?  If it's meaningful that it's the "new version" or something, could we instead say "Runs Lighthouse >= 5.6 ..." everywhere?
